README.md

```diff
@@ -85,7 +85,7 @@ Add dependencies (you can also add other modules that you need):
 ```xml
 <dependency>
     <groupId>org.jetbrains.kotlinx</groupId>
-    <artifactId>kotlinx-coroutines-core</artifactId>
+    <artifactId>kotlinx-coroutines-core-jvm</artifactId>
```

Comment:
In my opinion, `-jvm` is visual noise in this case and doesn't actually mean anything. Of course it's JVM, we're mentioning a Kotlin project in a Maven file. I understand that there's an extra dependency resolution step, but does it have any non-negligible effect?

Reply:
> I understand that there's an extra dependency resolution step, but does it have any non-negligible effect?

Just trying to make the world _better_ by saving ~130KB of networking traffic for each resolution when the artifact is not cached locally yet. :)